custom/icds_reports/ucr/data_sources/ag_care_monthly.json

```diff
@@ -0,0 +1,715 @@
+              "type": "boolean_expression",
+              "expression": {
+                "datatype": "string",
+                "type": "property_path",
+                "property_path": [
+                  "form",
+                  "consuming_ifa"
+                ]
+              },
+              "property_value": "yes"
+            }
+          },
+          "map_expression": {
+            "datatype": "integer",
+            "type": "property_path",
+            "property_path": [
+              "form",
+              "ifa_tablets_consumed"
+            ]
+          }
+        }
+      },
+      "parent_id": {
+        "type": "property_name",
+        "property_name": "host_person_id",
+        "datatype": null
+      }
+    },
+    "named_filters": {
+      "open_in_month": {
+        "type": "and",
+        "filters": [
+          {
+            "operator": "gte",
+            "type": "boolean_expression",
+            "expression": {
+              "type": "diff_days",
+              "from_date_expression": {
+                "type": "root_doc",
+                "expression": {
+                  "datatype": "date",
+                  "type": "property_name",
+                  "property_name": "opened_on"
+                }
+              },
+              "to_date_expression": {
+                "type": "named",
+                "name": "month_end"
+              }
+            },
+            "property_value": 0,
+            "comment": null
+          },
+          {
+            "type": "or",
+            "filters": [
+              {
+                "operator": "eq",
+                "type": "boolean_expression",
+                "expression": {
+                  "type": "root_doc",
+                  "expression": {
+                    "datatype": null,
+                    "type": "property_name",
+                    "property_name": "closed"
+                  }
+                },
+                "property_value": false,
+                "comment": null
+              },
+              {
+                "operator": "gte",
+                "type": "boolean_expression",
+                "expression": {
+                  "type": "diff_days",
+                  "from_date_expression": {
+                    "type": "named",
+                    "name": "month_start"
+                  },
+                  "to_date_expression": {
+                    "datatype": "date",
+                    "type": "property_name",
+                    "property_name": "closed_on"
+                  }
+                },
+                "property_value": 0,
+                "comment": null
+              }
+            ]
+          }
+        ]
+      }
+    },
+    "engine_id": "icds-ucr-citus",
+    "sql_settings": {
+      "citus_config": {
+        "distribution_type": "hash",
+        "distribution_column": "supervisor_id"
+      },
+      "primary_key": ["supervisor_id","doc_id"]
+    },
+    "sql_column_indexes": [
```

Comment:
Based this on [ccs_record_cases_monthly_v2](https://github.com/dimagi/commcare-hq/blob/055e67f2ace26419a03bc47ebc1a22f78ed2e4e8/custom/icds_reports/ucr/data_sources/ccs_record_cases_monthly_v2.json#L2751-L2775)